matlabMeas/MeasurementUtilities/sweep2DMeasSR830_Func.m

```diff
@@ -1,14 +1,20 @@
-function [avgmags] = sweep2DMeasSR830_Func(sweepParams1,sweepParams2,timeBetweenPoints,repeat,readSR830)
-%plotHandle = initializeSR830Meas2D_Func(sweepType,doBackAndForth);
+function [avgmags] = sweep2DMeasSR830_Func(sweepTypes, starts, stops, deltaParams, devices, portss, timeBetweenPoints,repeat,readSR830)
 
-sweepParams1cell = num2cell(sweepParams1);
-sweepParams2cell = num2cell(sweepParams2);
+%% Test command
+% sweep2DMeasSR830_Func({'Freq', 'ST'}, {1000, 0}, {10000, 1}, {1000, 0.1}, {SR830,SR830}, {{'Freq'},{'1'}}, 0.5, 5, SR830)
 
+plotHandle = initializeSR830Meas2D_Func(sweepTypes, starts, stops, deltaParams);
 
+haveDAC = 0;
```

Comment:
get rid of this. I'm guessing it was for testing purposes.

Reply:
COMMENT REQUESTED

Correct, can do.

With that said, would it be possible to include a 'smart haveDAC' that reads the instrumentList file to find if there is a dac and, therefore, only refresh it if need be?

Reply:
I think the best thing to do here is to probably make a text file that keeps track of open GUIs. We can do this on the initialization. Closing GUIs can also remove the GUI from the text file.